hbase-client/src/main/java/org/apache/hadoop/hbase/zookeeper/ZNodePaths.java

```diff
@@ -69,10 +69,25 @@ public class ZNodePaths {
   // znode used for log splitting work assignment
   public final String splitLogZNode;
   // znode containing the state of the load balancer
+  /**
+   * @deprecated Since 2.6.0, will be removed in 4.0.0. We use master local region to store this
```

Comment:
this class is IA.Private. Do we need a long deprecation period?

Reply:
We need to keep them here for migration.